### DISCUSSION TITLE
Error opening terminal: xterm-ghostty

### DISCUSSION THREAD

**Discussion by saeedvaziry** [ghostty-org/ghostty#3972] [Q&A] [answered]
I am trying to use `nano` on an ubuntu server which I did SSH into it using Ghostty and getting this error

![CleanShot 2024-12-30 at 00 54 15@2x](https://github.com/user-attachments/assets/6dfea0c3-5f11-461c-8d2f-e552f70df1de)

**Comment by ag-stanciu** [comment, marked as answer]
https://ghostty.org/docs/help/terminfo#ssh
